### ISSUE TITLE
Add ingame leaderboards

### ISSUE THREAD

**Issue by rockuSSS** [ddnet/ddnet#6382]
Since a huge part of this gamemode is speedrunning, it could get some enhancements. My suggestion is to add actual speedrunning leaderboards instead of having it in chat (/top5team, for example). My idea would be like a tab on ESC menu that you could go to and there would be all the possible leaderboards that ddnet has such as /top5, /top5team and whatever.

**Comment by rockuSSS** [comment]
![possible](https://user-images.githubusercontent.com/127057109/222988311-86e60111-4155-421a-a922-68bfa4f396db.png)
Add the tab where the blue line is

**Comment by Robyt3** [comment]
I agree, using the chat to transfer this information feels really incomplete for a game. We should have ingame menus for this and also for points (CC #9098).

**Comment by AssassinTee** [comment]
I am **almost** willing to set this to accepting-patches. Possible candidate for a replacement for #11215, however I'd want both, a leaderboard and a moderator board
